stretch_core/stretch_core/command_groups.py

```diff
@@ -538,6 +595,14 @@ def update(self, commanded_joint_names, invalid_joints_callback, **kwargs):
 
         return True
 
+    def set_trajectory_goals(self, points, robot):
+        if self.active_translate_mobile_base:
+            for t, x, v, a in get_waypoints(points, self.index_translate_mobile_base):
+                robot.base.trajectory.add_translate_waypoint(t_s=t, x_m=x, v_m=v, a_m=a)
+        elif self.active_rotate_mobile_base:
+            for t, x, v, a in get_waypoints(points, self.index_rotate_mobile_base):
+                robot.base.trajectory.add_rotate_waypoint(t_s=t, x_r=x, v_r=v, a_r=a)
+
```

Comment:
Let's replace the base waypoints interface with the multidof message's twist component. We should issue these twists using the base.set_velocity interface. For example:
```
base.set_velocity(twist.linear.x, twist.angular.z)
```

Reply:
I agree, but I'm going to do that in a separate PR